Sources/KlaviyoSwift/StateManagement.swift

```diff
@@ -165,12 +162,14 @@ struct KlaviyoReducer: ReducerProtocol {
             if let externalId = state.externalId {
                 initialState.externalId = externalId
             }
-            let queuedRequests = state.queue
-            initialState.queue += queuedRequests
+            // For any requests that get added between initilizing and initilized.
+            // Ex: when the app is invoked from a push notification after being killed from the app switcher.
+            let pendingRequests = state.pendingRequests
```

Comment:
This is the line that actually fixes the issue. When we don't have this, the requests that are added to `state.pendingRequests` after the initialization action were erased from state without enqueuing them.